github/gen-accessors.go

```diff
@@ -356,8 +369,8 @@ func ({{.ReceiverVar}} *{{.ReceiverType}}) Get{{.FieldName}}() *{{.FieldType}} {
   }
   return {{.ReceiverVar}}.{{.FieldName}}
 }
-{{else if .MapType}}
-// Get{{.FieldName}} returns the {{.FieldName}} map if it's non-nil, an empty map otherwise.
+{{else if or .MapType .ArrayType }}
+// Get{{.FieldName}} returns the {{.FieldName}} {{if .MapType}}map{{else if .ArrayType }}array{{end}} if it's non-nil, an empty map otherwise.
```

Comment:
```suggestion
// Get{{.FieldName}} returns the {{.FieldName}} {{if .MapType}}map{{else if .ArrayType }}slice{{end}} if it's non-nil, {{if .MapType}}an empty map{{else if .ArrayType }}nil{{end}} otherwise.
```